main.go

```diff
@@ -28,36 +27,48 @@ import (
 const (
 	pollInterval                = 15 * time.Second
 	pollTimeout                 = 3600 * time.Second
+	sourceVM             string = "vm"
```

Comment:
Please try to use variables defined in kubevirt package, do not define them by yourself

Reply:
There is no such kind for VMSnapshot, so I'd prefer to keep it as a string.